test/junit/scala/collection/immutable/VectorTest.scala

```diff
@@ -439,21 +439,17 @@ class VectorTest {
     val (pre, suf) = v.splitAt(1)
     val v2 = suf.prependedAll(pre)
     validateDebug(v2)
-    // Validation failed: assertion failed: sum of slice lengths (1024) at prefix2 should be equal to prefix length (1023)
-    // [error] Vector3(lengths=[32, 1023, 9215, 9984, 10000])
     assertEquals(10000, v.last)
-    assertEquals(10000, v2(v2.length-1)) // ArrayIndexOutOfBoundsException
+    assertEquals(10000, v2(v2.length-1))
```

Comment:
I like to keep a comment showing the previous behavior, because otherwise later you look at a test and don't know what it is testing. In particular, asserting that it did not throw.

Reply:
Good to know. I wasn't sure about these comments. They are now restored.